### ISSUE TITLE
search using both cursorMark and timeAllowed

### ISSUE THREAD

**Issue by migueldvb** [andycasey/ads#94]
Simple searches using SearchQuery return an `APIResponseError` `org.apache.solr.common.SolrException: Can not search using both cursorMark and timeAllowed`

## Expected Behavior
```
>>> import ads


>>> papers = ads.SearchQuery(q="supernova", sort="citation_count")

>>> for paper in papers:
>>>    print(paper.title)
```
## Current Behavior
```
APIResponseError: u'{"responseHeader":{"status":400,"QTime":15,"params":{"q":"supernova","fl":"id,author,first_author,bibcode,id,year,title","cursorMark":"*","sort":"citation_count desc,id desc","rows":"50","wt":"json"}},"error":{"metadata":["error-class","org.apache.solr.common.SolrException","root-error-class","org.apache.solr.common.SolrException"],"msg":"Can not search using both cursorMark and timeAllowed","code":400}}\n'
```

## Your Environment
This error occurs with python 2.7 and 3.6

* ads version used (hint: `python -c "import ads; print(ads.__version__)"`)
0.12.2

**Comment by vsudilov** [comment]
Thanks for the report, will look into a fix. In the meantime you can specify a start kwarg and it should work.

**Comment by vsudilov** [comment]
Looks like the behaviour was introduced upstream @ https://github.com/romanchyla/montysolr/commit/ca2d42324c2875634863557cdc03aafd874897d3

@aaccomazzi , this change breaks this client's functionality -- should we revert cursorMark?

**Comment by aaccomazzi** [comment]
Ugh, I didn't see this coming...  The reasoning behind the error is explained here: http://lucene.472066.n3.nabble.com/cursorMark-and-timeAllowed-are-mutually-exclusive-td4214685.html
@romanchyla what do you think we should do?  The cursorMark implementation is more efficient and robust than offset queries, but removing the timeout is def not ideal.

**Comment by andycasey** [comment]
Is there some way that they can be used in combination?

Andy

**Comment by aaccomazzi** [comment]
Not according to the documentation.  Seems to be a somewhat anal choice made to guarantee the presence of a token no matter what.

**Comment by romanchyla** [comment]
We could enforce the timeallowed in our api (unless token is present, and
token will be there only after the first page of results, I think....)

**Comment by aaccomazzi** [comment]
So I suggest we remove timeAllowed till we sort this out on our end, otherwise the client library is broken.

**Comment by vsudilov** [comment]
Cool, if you need us to make any changes related to this in the future please reopen or create a new issue
